src/themes/styled.d.ts

```diff
@@ -11,6 +11,10 @@ declare module 'styled-components' {
 			readonly primary: string
 			readonly secondary: string
 			readonly link: string
+
+			// colours for validation (used for example in forms)
```

Comment:
May I suggest better comment formatting? :grimacing:
```suggestion
			// Colors for validation (used for example in forms)
```